ocs_ci/ocs/scale_lib.py

```diff
@@ -709,3 +709,84 @@ def increase_pods_per_worker_node_count(pods_per_node=500, pods_per_core=10):
     # During manual execution observed each node took 240+ sec for update
     timeout = machine_count * 300
     utils.wait_for_machineconfigpool_status(node_type=constants.WORKER_MACHINE, timeout=timeout)
+
+
+def construct_pvc_creation_yaml_bulk_for_kube_job(no_of_pvc, access_mode, sc_name):
+    """
+    Function to construct pvc.yaml to create bulk of pvc's using kube_job
+
+    Args:
+        no_of_pvc(int): Bulk PVC count
+        access_mode (str): PVC access_mode
+        sc_name (str): SC name for pvc creation
+
+    Returns:
+         pvc_dict_list (list): List of all PVC.yaml dicts
+
+    """
+    if access_mode == 'ReadWriteMany' and 'rbd' in sc_name:
```

Comment:
Only for RBD_RWX we need to add `pvc_data['spec']['volumeMode'] = 'Block'` in pvc.yaml due to this am setting a flag but let me use this check directly in loop. 
the sc_name is expected in function call, it can be cephfs/rbd due to this not using constant.XXXXXX here, also `'rbd' in sc_name` validates if the sting contains 'rbd' in it.

Reply:
Will change the string in check like `if access_mode == 'ReadWriteMany' and sc_name == constants.CEPHBLOCKPOOL:`